WF-507:Dogfooding fixes (#7)
spatialos/FlexibleProjectExample: docs/reference/client-worker-configuration.md

      "radius": 2
    }
  },
  "streamingQuery": [
    ...
  ],
  "componentDelivery": {
    "default": "RELIABLE_ORDERED",
    "checkoutAllInitially": true
  },
  "componentSettings": {
    ...
  },

>   "permissions": [

Reviewer: I don't think the permissions field is supposed to be an array

Author: This is the format specified in our current docs: https://docs.improbable.io/reference/13.5/shared/worker-configuration/permissions#worker-permissions

Author: Fixed. :)

    ...
  ],
  "uploadConfiguration": {
    "windows": {
      "localBuildDirectory": "C:\project\bin\windows"
    },
    "macos": {
      ...
    },
    "linux": {
      ...
    }
  }
}
```

| Field | Required/Optional | Description | 
| :------------- | :------------- | :------- |

> | `workerType` | Required | Used to identify this worker type elsewhere in your project. <br><br> **Note:** A `workerType` must be unique project-wide between server and client workers. The `workerType` specified in this field must also match the `workerType` that your worker reports in its `ConnectionParameters` when connecting to a deployment. For more infromation, see the [API reference](https://docs.improbable.io/reference/latest/csharpsdk/api-reference#improbable-worker-connectionparameters-class). |

Reviewer 2: infromation -> information

Author: Fixed. :)

    "windows": {
      "localBuildDirectory": "C:\project\bin\windows"
    },
    "macos": {
      ...
    },
    "linux": {
      ...
    }
  }
}
```

| Field | Required/Optional | Description | 
| :------------- | :------------- | :------- |

> | `workerType` | Required | Used to identify this worker type elsewhere in your project. <br><br> **Note:** A `workerType` must be unique project-wide. The `workerType` specified in this field must also match the `workerType` that your worker instance reports in its `ConnectionParameters` when connecting to a deployment. For more information, see the [API reference](https://docs.improbable.io/reference/latest/csharpsdk/api-reference#improbable-worker-connectionparameters-class). |

Reviewer 3: thanks for clarifyng this!

> | `workerType` | Required | Used to identify this worker type elsewhere in your project. <br><br> **Note:** A `workerType` must be unique project-wide between server and client workers. The `workerType` specified in this field must also match the `workerType` that your worker reports in its `ConnectionParameters` when connecting to a deployment. For more information, see the [API reference](https://docs.improbable.io/reference/latest/csharpsdk/api-reference#improbable-worker-connectionparameters-class). |

Reviewer 4: I'm not sure what "unique project-wide between server and client workers" means - is it that within a project, you can only use a particular `workerType` once (so if you use it for a client-worker, you can't use it for a server-worker)? If so I'd maybe leave out "between server and client workers" and just stick with "unique project-wide". 

I've also changed "that your worker reports" to that your worker instance reports" as I _think_ that is more accurate?

```suggestion
| `workerType` | Required | Used to identify this worker type elsewhere in your project. <br><br> **Note:** A `workerType` must be unique project-wide between server and client workers. The `workerType` specified in this field must also match the `workerType` that your worker instance reports in its `ConnectionParameters` when connecting to a deployment. For more information, see the [API reference](https://docs.improbable.io/reference/latest/csharpsdk/api-reference#improbable-worker-connectionparameters-class). |
```

Author: Removed the "between server and client workers" part. Added "instance".

> |`layer`| Optional | The name of the simulation layer of this worker, used to determine the load balancing strategy. For more information, see [introduction to layers](https://docs.improbable.io/reference/latest/shared/worker-configuration/layers#introduction-to-layers). |

Reviewer 4: ```suggestion
|`layer`| Optional | The name of the simulation layer of this worker type, used to determine the load balancing strategy. For more information, see [Layers](https://docs.improbable.io/reference/latest/shared/worker-configuration/Layers). |
```

Updated the link because I changed this yesterday with the release of Kraken :)

Author: Nice. Done!

| `streamingQuery` | Optional | Specifies the streaming queries the bridge will subscribe to. Analogous to [`streaming_query` in the previous configuration version](https://docs.improbable.io/reference/latest/shared/worker-configuration/bridge-config#streaming-queries). |
| `componentDelivery` | Optional | Specifies the delivery settings for the worker's components. Analogous to [`component_delivery` in the previous configuration version](https://docs.improbable.io/reference/latest/shared/worker-configuration/bridge-config#component-delivery). |
| `componentSettings` | Optional | Defines component specific settings. Analogous to [`component_settings` in the previous configuration version](https://docs.improbable.io/reference/latest/shared/worker-configuration/bridge-config#component-settings). |
| `permissions` | Optional | Specifies the worker's entity permissions. Analogous to [`permissions` in the previous configuration version](https://docs.improbable.io/reference/latest/shared/worker-configuration/permissions). |

> | `uploadConfiguration` | Optional | Specifies the client worker's build directory for each platform. If provided, the `spatial alpha cloud upload` CLI command will upload the given client worker enabling it to be used with the [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher). <br><br> It contains three optional fields (`windows`, `macos`, `linux`) which each contain one required `localBuildDirectory` string field. The `localBuildDirectory` field specifies the directory containing the worker binary to be executed. <br><br> **Note:** Your built out worker must be self-contained to the `localBuildDirectory` and all files that your worker might depend on while running must be present in this folder. Files outside of this folder will not be uploaded to the cloud as part of your project assembly. |

Reviewer 4: ```suggestion
| `uploadConfiguration` | Optional | Specifies the client worker's build directory for each platform. If provided, the `spatial alpha cloud upload` CLI command uploads the given client worker, enabling you to use it with the [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher). <br><br> It contains three optional fields (`windows`, `macos`, `linux`) which each contain one required `localBuildDirectory` string field. The `localBuildDirectory` field specifies the directory containing the worker binary to be executed. <br><br> **Note:** Your built-out worker must be self-contained in the `localBuildDirectory` and all files that your worker might depend on while running must be in this folder. Files outside this folder will not be uploaded to the cloud as part of your project assembly. |
```

Author: Done.
